Backend/app/loan_management/book_loans_service.py

```diff
@@ -1,4 +1,5 @@
 from enum import auto
+import json
```

Comment:
I don't think you're using this library? Consider deleting it if you imported it by mistake or if you imported it and then realized you didn't need it.